src/package/nuspec/Microsoft.TestPlatform.Portable.nuspec

```diff
@@ -34,6 +34,7 @@
     <file src="net451\$Runtime$\Microsoft.VisualStudio.TestPlatform.Client.dll"	target="tools\net451\Microsoft.VisualStudio.TestPlatform.Client.dll" />
     <file src="net451\$Runtime$\Microsoft.VisualStudio.TestPlatform.Common.dll"	target="tools\net451\Microsoft.VisualStudio.TestPlatform.Common.dll" />
     <file src="net451\$Runtime$\Microsoft.VisualStudio.TestPlatform.ObjectModel.dll"	target="tools\net451\Microsoft.VisualStudio.TestPlatform.ObjectModel.dll" />
+    <file src="net451\$Runtime$\Microsoft.VisualStudio.Coverage.CoreLib.Net.dll"	target="tools\net451\Microsoft.VisualStudio.Coverage.CoreLib.Net.dll" />
```

Comment:
We should clean up old libs from packages. Also from vsix and test vsix in VS and test platform nuget package

Reply:
Done.